cardano-cli/golden/files/golden/help/shelley_governance_action_create-new-committee.cli

```diff
@@ -0,0 +1,15 @@
+Usage: cardano-cli shelley governance action create-new-committee --epoch EPOCH
+            --drep-cold-verification-key-file FILE
+            [--cost-model-file FILE]
+            --out-file FILE
+
+  Create a new-committee action
+
+Available options:
+  --epoch EPOCH            The epoch number in which the update proposal is
```

Comment:
We don't need --epoch; the user will not set an epoch, they just throw it out and it lands when it lands. There will be a protocol parameter that deals with deadlines.